pkgs/data/icons/posy-cursors/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, stdenvNoCC, fetchgit }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "posy-cursors";
+  version = "1.6";
+
+  src = fetchgit {
+    url = "https://github.com/simtrami/posy-improved-cursor-linux";
+    rev = "refs/tags/${version}";
+    sha256 = "sha256-i0N/QB5uzqHapMCDl6h6PWPJ4GOAyB1ds9qlqmZacLY=";
+  };
+
+  installPhase = ''
+    install -dm 755 $out/share/icons
+    cp -r Posy_Cursor* $out/share/icons
+  '';
+
+  meta = with lib; {
+    description = "Posy's Improved Cursors for Linux";
+    homepage = "https://github.com/simtrami/posy-improved-cursor-linux";
+    platforms = platforms.unix;
+    maintainers = with maintainers; [ mkez ];
```

Comment:
Also, we need to know the license of the project. I cant find any on the repo site.

Reply:
Sadly the original creator has not specified a license: [http://www.michieldb.nl/other/cursors/](http://www.michieldb.nl/other/cursors/).
Other packages distributing this same cursor theme are also don't have a license specified: [AUR](https://aur.archlinux.org/packages/posy-improved-cursors), [opendestktop](https://www.opendesktop.org/p/1795042).

Reply:
I would say that is a deal breaker then. We don't know if we can redistribute it. Maybe you can ask on the issue tracker of that project for the license?

Reply:
It is then by default `unfree`